fixtures/gists-app/app/routes/__layout.tsx

```diff
@@ -1,10 +1,16 @@
+import * as React from "react";
 import { Outlet } from "remix";
 
+export interface ContextData {
+  nested: boolean;
+}
+
 export default function LayoutTest() {
+  let [contextData] = React.useState<ContextData>({ nested: true });
```

Comment:
```suggestion
  const [contextData] = React.useState<ContextData>({ nested: true });
```

Reply:
We only `const`-ify externally-facing code. We `let` it out internally though :)